src/main/java/org/myrobotlab/service/config/VertxConfig.java

```diff
@@ -0,0 +1,9 @@
+package org.myrobotlab.service.config;
+
+public class VertxConfig extends ServiceConfig {
```

Comment:
why are there 2 vertxconfigs, one with a capital X and one with a lower case x ?

Reply:
removed the incorrect one